packages/cf.js/src/provider.ts

```diff
@@ -16,6 +17,21 @@ import {
   UpdateStateEventData
 } from "./types";
 
+export const jsonRpcMethodNames = {
```

Comment:
Why do we even need this mapping? Why isn't each key actually defined as the value in the `Node.MethodName` enum?

Reply:
I didn't want to alter any of the current messaging layer. There's actually en enum called `Node.RpcMethodName` to leave the original one untouched. If you prefer to replace the old names with the new ones, just let me know.